services/migrations/gogs_test.go

```diff
@@ -99,8 +99,8 @@ func TestGogsDownloadRepo(t *testing.T) {
 			Content:     "test",
 			Milestone:   "",
 			State:       "open",
-			Created:     time.Date(2019, 06, 11, 8, 16, 44, 0, time.UTC),
-			Updated:     time.Date(2019, 10, 26, 11, 07, 2, 0, time.UTC),
+			Created:     time.Date(2019, 0o6, 11, 8, 16, 44, 0, time.UTC),
```

Comment:
this could be bugs!

Reply:
Yup that's a bug in gofumpt, at least it's not expected to have octal heres.

Reply:
https://github.com/mvdan/gofumpt#added-rules

> Octal integer literals should use the 0o prefix on modules using Go 1.13 and later
example
const perm = 0755
const perm = 0o755

Reply:
Yeah, but they should IMO only be used on such integers, and not with other non-octal digits, as this makes this argument inconsistent with the others.

Reply:
I changed them to be just normal integer ...